includes/functions-upgrade.php

```diff
@@ -403,7 +403,7 @@ function yourls_alter_url_table_to_14_part_two() {
  *
  */
 function yourls_update_table_to_14() {
-    $ydb = yourls_get_db();
+    $ydb = yourls_get_db('read-update_table_to_14');
```

Comment:
The potential `UPDATE` queries below are pretty `write-`y.

```suggestion
    $ydb = yourls_get_db('write-update_table_to_14');
```

Note also L428 has a couple typos that I can't make suggestions for ("Huho" -> "Uh-oh"; "rown" -> "row") since they're too far from the patch. (Dumb, arbitrary limitation IMO.)